services/registration/src/routes/statistics.ts

```diff
@@ -2,98 +2,116 @@
 import { asyncHandler } from "@api/common";
 import express from "express";
 
-import { ApplicationModel } from "../models/application";
+import { ApplicationModel, StatusType } from "../models/application";
 import { BranchModel, BranchType } from "../models/branch";
 
 export const statisticsRouter = express.Router();
 
 statisticsRouter.route("/").get(
   asyncHandler(async (req, res) => {
-    let totalUsers = 0;
-    let confirmedUsers = 0;
-    const submittedUsers = await ApplicationModel.countDocuments({
-      applicationSubmitTime: { $exists: true, $ne: null },
-    });
```

Comment:
Also, this is something that wasn't in last time, but we aren't getting statistics by hexathon. This just compiles information for every hexathon. So you'll need to add some way to filter by hexathon that needs to be required. Prob by search query `?`.